ember-addon-main.js

```diff
@@ -69,7 +72,7 @@ module.exports = {
   htmlbarsOptions: function() {
     var projectConfig = this.projectConfig() || {};
     var EmberENV = projectConfig.EmberENV || {};
-    var templateCompilerPath = this.templateCompilerPath() + '.js';
```

Comment:
hmm `this.templateCompilerPath()` now returns (sometimes with '.js')

I can add a `ensureEndsInJs()` or something